lumberjack.go

```diff
@@ -205,6 +211,13 @@ func (l *Logger) openNew() error {
 			return fmt.Errorf("can't rename log file: %s", err)
 		}
 
+		// Compress the backup if enabled
+		if l.CompressBackups == true {
+			if err := compressLog(newname); err != nil {
```

Comment:
The problem with this is that a simple log statement is now going to be blocked on gzipping a multi megabyte log file.  This should probably be run in a goroutine so it won't block the program.... probably making this part of the deleteAll method (and renaming it to cleanup() or something) would be a good idea.